docs/specification.md

```diff
@@ -0,0 +1,155 @@
+# Homebase Lite Spec Draft
+
+This spec draft is for the proposed Tezos smart contract to be used as the on-chain component of a new lightweight Homebase tool.
+
+# Requirements
+
+This is intended to be a simpler option than baseDAO, both in complexity and usage, for communities to request feedback, run polls, etc.
+
+In summary, the smart contract should support:
+
+- Non-binding proposals based on an IPFS document.
+- Voting from a set of choices per proposal.
+- Proposal's life-cycles based on time and submitted votes.
+- Multiple roles: admin, maintainer(s) and participant(s)
+- Configuration options that can be modified by maintainers.
+- An external “governance token” in the form of an existing FA2-compliant contract.
+
+more details can be found below.
+
+As usual, this smart contract will be covered by unit and property tests.
+If one or more active `TZIP`s can apply to it, support for them will be added.
+
+Documentation on the options, the origination process and tests will be included.
+
+Note: there are no restrictions on the programming language to be used for development, nor a typescript API should be needed.
```

Comment:
I would remove all these lines. I put them in the draft to be discussed/clear but they don't really belong here anymore:
1. testing should be documented elsewhere
2. TZIP(s) should have their own section here (or in a dedicated doc) once they are supported
3. documentation and programming language are also promises that don't need to be repeated IMO